drivers/dose/include/dose_params.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2019 Juergen Fitschen <me@jue.yt>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+
+/**
+ * @ingroup     drivers_dose
+ * @{
+ * @file
+ * @brief       Default configuration for the Differentially Operated Serial Ethernet driver
+ *
+ * @author      Juergen Fitschen <me@jue.yt>
+ */
+
+#ifndef DOSE_PARAMS_H
+#define DOSE_PARAMS_H
+
+#include "board.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Set default configuration parameters for the DOSE driver
+ * @{
+ */
+#ifndef DOSE_PARAM_UART
+#define DOSE_PARAM_UART         (UART_DEV(0))
+#endif
+#ifndef DOSE_PARAM_BAUDRATE
+#define DOSE_PARAM_BAUDRATE     (115200)
+#endif
+#ifndef DOSE_PARAM_SENSE_PIN
+#define DOSE_PARAM_SENSE_PIN    (GPIO_PIN(0, 0))
```

Comment:
```suggestion
#define DOSE_PARAM_SENSE_PIN    GPIO_UNDEF
```
If you want to signal that pin can be unused by default.
Beware that calling gpio functions on a GPIO with the value `GPIO_UNDEF` will lead to crashes on some platforms. (`gpio_set()` and friends are not expected to do sanity checking of parameters for the sake of responsiveness)

Reply:
I would suggest to make the `SENSE_PIN` optional in a follow-up PR. Otherwise the number of tests that must be performed nearly doubles. But I'll keep that in mind ;)